README.md

```diff
@@ -60,20 +60,104 @@ A CPU runtime that takes advantage of sparsity within neural networks to reduce
+
+## Installation Guide
+
+- [Hardware Support and System Requirements](https://docs.neuralmagic.com/deepsparse/source/hardware.html)
+- DeepSparse Community Edition
+- DeepSparse Enterprise Edition
+
 ## Features
 
 - 🔌 [DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server)
 - 📜 [DeepSparse Benchmark](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark)
 - 👩‍💻 [NLP and Computer Vision Tasks Supported](https://github.com/neuralmagic/deepsparse/tree/main/examples)
-- 🧰 [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html)
 
-## Installation
-The DeepSparse Engine is tested on Python 3.6-3.10, ONNX 1.5.0-1.12.0, ONNX opset version 11+, and manylinux compliant. Using a [virtual environment](https://docs.python.org/3/library/venv.html) is highly recommended. Install the engine using the following command:
+## Installation Guide
+
+## 🧰 Hardware Support and System Requirements
+
+Review [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html) to understand system requirements. The DeepSparse Engine works natively on Linux; Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.
+
+
+## Installation Guide
+The DeepSparse Engine is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, ONNX opset version 11+, and manylinux compliant. Using a [virtual environment](https://docs.python.org/3/library/venv.html) is highly recommended. 
+
+## DeepSparse Community Edition
+
+Install the Community Edition as follows: 
 
 ```bash
 pip install deepsparse
 ```
 
+To trial or inquire about licensing for DeepSparse Enterprise Edition, see the following DeepSparse Enterprise section. 
+
+## DeepSparse Enterprise Edition
+
+Install the Enterprise Edition as follows:
+
+```bash
+pip install deepsparse-ent
+```
+
+The DeepSparse Enterprise Edition requires a valid license to run the engine and can be licensed for production, commercial applications. There are two options available:
+
+### 90-Day Enterprise Trial License
+To try out the DeepSparse Enterprise Edition and get a Neural Magic Trial License, complete our [registration form](https://neuralmagic.wpengine.com/deepsparse-engine-free-trial/). Upon submission, the license will be emailed immediately to you and your 90-day term starts right then.
+
+### Enterprise Edition License
+To learn more about DeepSparse Enterprise Edition pricing, [review our pricing page](https://neuralmagic.wpengine.com/deepsparse-engine-free-trial/). Details on how to [contact our Sales team](https://neuralmagic.com/deepsparse-engine/#form) and discuss your use case further for a custom quote are also available.
+
+## DeepSparse Enterprise Edition License Management
```

Comment:
Maybe re-organize these sections as follow for better flow:

1. DeepSparse Enterprise Edition
    1. Getting a License
        1.  90-day Enterprise Trial License
        2. Enterprise Edition License
    2. Installing a license
    3. Validating a license

```markdown
## DeepSparse Enterprise Edition

### Getting a license
#### 90-day Enterprise Trial License
#### Enterprise Edition License

### Installing a license
### Validating a license
```